pkg/cli/environment.go

```diff
@@ -109,6 +113,8 @@ func New() *EnvSettings {
 		RepositoryCache:           envOr("HELM_REPOSITORY_CACHE", helmpath.CachePath("repository")),
 		BurstLimit:                envIntOr("HELM_BURST_LIMIT", defaultBurstLimit),
 		QPS:                       envFloat32Or("HELM_QPS", defaultQPS),
+		MaxChartSize:              envInt64Or("HELM_MAX_CHART_SIZE", 0),
+		MaxFileSize:               envInt64Or("HELM_MAX_FILE_SIZE", 0),
```

Comment:
```suggestion
		MaxChartFileSize:               envInt64Or("HELM_MAX_CHART_FILE_SIZE", 0),
```

IMHO, terminology (here and other places) should use the more specific "max chart file size". Which is what this functionality applies to. "max file size" would seem to apply to any file.

Reply:
That said, for Helm 4 (the `main` branch). We can probably remove this "max chart file size" functionality all together. It was originally introduced to prevent https://github.com/helm/helm/security/advisories/GHSA-5xqw-8hwv-wg92 . But https://github.com/helm/helm/pull/30754 removed the usage of (the vulnerable) `github.com/xeipuuv/gojsonschema` library for JSON schema parsing.

I would propose continuing with the flag in this PR, so we can easily backport (via cherry-pick style PR to `dev-v3` branch). Then we can look to remove its usage for Helm 4.